### ISSUE TITLE
RCA Confluence - Pinned Messages - No Spaces or Punctuation

### ISSUE THREAD

**Issue by mattpminer** [echoboomer/incident-bot#365]
Seeing behavior in which the pinned messages in a slack channel are persisted correctly, but when an RCA is created in confluence, all spaces and punctuation are taken out.

**In the incident page itself on incident bot web:**

```
Jack Johnson
27/10/2023 20:05:08 UTC
<https://github.com/namespace/repo/pull/4759> filed

Mary Ann
27/10/2023 20:55:21 UTC
PR is tested and validated locally, merged to main. After it builds we will deploy to pluto
```

**In the RCA Confluence page:**

```
Pinned Messages
These messages were pinned during the incident by users in Slack.
This information is useful for establishing the incident timeline and providing diagnostic data.
Jack Johnson @ 27/10/2023 20:01:35 UTC -  httpsgithubcomnamespacerepopull4759filed
Mary Ann @ 27/10/2023 20:55:21 UTC -  PRistestedandvalidatedlocallymergedtomainAfteritbuildswewilldeploytopluto
```

**Comment by echoboomer** [comment]
@mattpminer this is a carryover from when the frontend would break when displaying certain characters so they get stripped. Thanks for reporting, will take a look.

**Comment by echoboomer** [comment]
@mattpminer this has been resolved with release `v1.5.7` - you should be able to handle any pinned messages moving forward. If you run into any additional issues, please let me know.
